main/ui/src/main/java/org/cryptomator/ui/changepassword/ChangePasswordController.java

```diff
@@ -71,6 +69,7 @@ public void finish() {
 			CryptoFileSystemProvider.changePassphrase(vault.getPath(), MASTERKEY_FILENAME, oldPasswordField.getCharacters(), newPassword.get());
 			LOG.info("Successful changed password for {}", vault.getDisplayableName());
```

Comment:
I know this isn't yours to blame, but while your at it, let's change this to "Successfully" :wink:

Reply:
Yeah, that's better. I can change it this evening.